packages/fast-usdc/src/exos/advancer.js

```diff
@@ -291,8 +291,8 @@ export const prepareAdvancerKit = (
         notifyFacet: M.remotable(),
         borrowerFacet: M.remotable(),
         poolAccount: M.remotable(),
+        settlementAddress: ChainAddressShape,
         intermediateRecipient: M.opt(ChainAddressShape),
-        settlementAddress: M.opt(ChainAddressShape),
```

Comment:
good catch!

What prompted it, I wonder?

Reply:
I can't remember but I'm guessing some type feedback